api/pb/api.proto

```diff
@@ -92,6 +92,7 @@ message SetLogLevel {
 
 message AccountTxs {
     repeated string txs = 1;
+    uint64 validatedLayer = 2;
```

Comment:
validatedLayer is same for all requested accounts?

Reply:
No. If you call them one-by-one, it may advance in that time. If you want to store a single layer, you should store the minimal one you receive, since your code should deal with duplicates anyway, and that's all a too small layer will do - create duplicates.